sources/logreg_predict.py

```diff
@@ -0,0 +1,14 @@
+from Tools.LogReg import LogReg
+from Tools.ArgParser import ArgParser
+from Tools.DatasetHandler import DatasetHandler
+
+
+def run():
+    log_predict = LogReg(train=False)
+    log_predict.load_theta()
+    prediction = log_predict.predict(log_predict.np_df)
+    log_predict.write_to_csv(prediction, ["Hogwarts House"])

```

Comment:
If the low-level source code of write_to_csv is protected (try: except:) it's ok. See previous review comments about it.

Reply:
Also, the Hogwarts House in plain text mean the program isn't for generic data. But i've heard you are working on, cool.